libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/view/MapboxPrimaryManeuver.kt

```diff
@@ -7,6 +7,7 @@ import androidx.core.content.ContextCompat
 import com.mapbox.navigation.ui.maneuver.R
 import com.mapbox.navigation.ui.maneuver.model.ManeuverInstructionGenerator
 import com.mapbox.navigation.ui.maneuver.model.PrimaryManeuver
+import com.mapbox.navigation.ui.maneuver.model.RoadShield
 
```

Comment:
Noticed that we're still using `@JvmOverloads` here https://github.com/mapbox/mapbox-navigation-android/issues/4134